debezium-ddl-parser/src/test/resources/mysql/examples/dml_select.sql

```diff
@@ -322,4 +322,7 @@ FROM
     your_table,
     (SELECT @sum := 0) AS init
 ORDER BY
-    some_order_column;
\ No newline at end of file
+    some_order_column;
+
+-- statement is supported as the field name
+SELECT REPLACE(statement, ' ','') as statement from your_table;
```

Comment:
So I'm a bit confused, I added this to the MySQL `dml_select.sql` without modifying the grammar, and this works without an error, see this:
```
#begin
-- statement is supported as the field name
SELECT REPLACE(statement, ' ','') as statement from your_table;
#end
```
Did you check whether the DDL parser failed without making changes to the grammar?